simplq/src/components/pages/Home/index.jsx

```diff
@@ -10,6 +10,7 @@ const Home = (props) => {
       <HowToCreate />
       <HowToJoin />
       <ExtraInfo />
+      <Footer />
     </>
```

Comment:
@daltonfury42 do we want to show the footer only on the homepage or all pages? If it's all pages, the Footer should be added in the Routes.

Reply:
For now, let it be just for the homepage